src/script/auth/page/Index.test.tsx

```diff
@@ -72,6 +73,37 @@ describe('when visiting the index page', () => {
       .toBe(true);
   });
 
+  it('redirects to SSO login if default SSO code is set', async () => {
+    const history = createMemoryHistory();
+    const historyPushSpy = spyOn(history, 'push');
+
+    const defaultSSOCode = 'defaultt-a4b0-4c59-a31d-303a7f5eb5ab';
```

Comment:
```suggestion
    const defaultSSOCode = 'default-a4b0-4c59-a31d-303a7f5eb5ab';
```

Reply:
That's not a typo, a valid UUID v4 needs 8 chars in the first chunk